backend/src/migrations/performance_testing_queries.py

```
@@ -0,0 +1,745 @@
+
+            execution_time = (end_time - start_time) * 1000  # Convert to milliseconds
+
+            return {
+                'description': description,
+                'execution_time_ms': round(execution_time, 3),
+                'result_count': len(result) if hasattr(result, '__len__') else 1,
+                'success': True,
+                'timestamp': datetime.utcnow().isoformat()
+            }
+
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)